src/main/java/com/azure/cosmos/sample/ReadManyItems.java

```diff
@@ -76,178 +79,165 @@ public static void main(String[] args) {
             p.readManyItemsDemo();
             System.out.println("Demo complete, please hold while resources are released");
         } catch (Exception e) {
-            System.out.println("Cosmos getStarted failed with: "+ e);
+            System.out.println("Cosmos getStarted failed with: " + e);
         } finally {
             System.out.println("Closing the client");
             p.close();
         }
         System.exit(0);
     }
-
     // </Main>
 
     private void readManyItemsDemo() throws Exception {
-        System.out.println("Using Azure Cosmos DB endpoint: " +AccountSettings.HOST);
-
-        // Create sync client
-        // <CreateSyncClient>
+        System.out.println("Using Azure Cosmos DB endpoint: " + AccountSettings.HOST);
+        docs = generateDocs(NUMBER_OF_DOCS);
         client = new CosmosClientBuilder()
                 .endpoint(AccountSettings.HOST)
                 .key(AccountSettings.MASTER_KEY)
                 // Setting the preferred location to Cosmos DB Account region
                 // UK South is just an example. User should set preferred location to the Cosmos
                 // DB region closest to the application
                 .preferredRegions(Collections.singletonList("UK South"))
-                .consistencyLevel(ConsistencyLevel.EVENTUAL)
-                .buildClient();
-
-        createDatabaseIfNotExists();
-        createContainerIfNotExists();
+                .consistencyLevel(ConsistencyLevel.SESSION)
+                .buildAsyncClient();
+
+        Mono<Void> databaseContainerIfNotExist = client.createDatabaseIfNotExists(databaseName)
+                .flatMap(databaseResponse -> {
+                    database = client.getDatabase(databaseResponse.getProperties().getId());
+                    System.out.println("\n\n\n\nCreated database ReadManyItemsDB.\n\n\n\n");
+                    CosmosContainerProperties containerProperties = new CosmosContainerProperties(containerName, "/id");
+                    ThroughputProperties throughputProperties = ThroughputProperties.createManualThroughput(10000);
```

Comment:
should we use >10,000 so to create a container containers multiple partitions? probably can use 15,000

Reply:
good suggestion, changed.